net/mwan3/files/lib/mwan3/mwan3.sh

```diff
@@ -291,10 +297,31 @@ mwan3_set_connected_ipv4()
 
 }
 
-mwan3_set_connected_iptables()
+mwan3_set_connectd_ipv6()
 {
-	local connected_network_v6 source_network_v6
+	local connected_network_v6
+
+	for connected_network_v6 in $($IP6 route | awk '{print $1}' | egrep "$IPv6_REGEX"); do
+		$IPS -! add mwan3_connected_v6_temp "$connected_network_v6"
+	done
+	$IPS swap mwan3_connected_v6_temp mwan3_connected_v6
+	$IPS destroy mwan3_connected_v6_temp
+
+	if [ "$FIX_KERNEL_ACTUAL_SK_BUG" = "yes" ]; then
```

Comment:
Use natural sh booleans:

```suggestion
	if "$FIX_KERNEL_ACTUAL_SK_BUG"; then
```